shalonday/theonlinebrainproject
# Add delete modal

Changes:
 - Add DeleteModal based on MUI AlertDialog
 - Confirming DELETE removes the node from the displayed current tree

To test:
- Try deleting a node. After confirming DELETE the displayed tree should no longer include the deleted node.
